webapp/src/main/java/org/opentestsystem/rdw/admin/model/CsvSubject.java

```diff
@@ -0,0 +1,19 @@
+package org.opentestsystem.rdw.admin.model;
+
+/**
+ * This class represents the Group subjects allowed during CSV Group uploads.
+ */
+public enum CsvSubject {
```

Comment:
Except that All != null.  If no subject is specified then (All/Null) is assumed, and can be set to Math/ELA by the next record.  If All is specified, and the next record sets the subject to Math/ELA, that's a validation failure.  So we do actually have 4 "acceptable" subject values: null, All, Math, and ELA.